### PR TITLE
Namespace for Context?

### PR DESCRIPTION
**Pull request by cijothomas**
Opening a PR to seek opinions.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Api/Trace/ActivityContextExtensions.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 using System.Diagnostics;
 
-namespace OpenTelemetry.Context.Propagation
+namespace OpenTelemetry.Trace
```
Comment (cijothomas) [inline src/OpenTelemetry.Api/Trace/ActivityContextExtensions.cs:19]: `ActivityContext` does not expose `IsValid` as required by spec. So this extension method is compensating for that. Question is: What namespace should this use? 
I changed to OpenTelemetry.Trace as everything else required for Tracing/Activity is in that namespace. But the context/propagation classes who require this are in OpenTelemetry.Context.Propagation ns..

Comment (reyang) [reply src/OpenTelemetry.Api/Trace/ActivityContextExtensions.cs:19]: In this case - `OpenTelemetry`.

File: src/OpenTelemetry.Api/Context/Propagation/B3Propagator.cs [hunk 2]
```diff
@@ -20,6 +20,7 @@
 using System.Linq;
 using System.Text;
 using OpenTelemetry.Internal;
+using OpenTelemetry.Trace;
```
Comment (reyang) [inline src/OpenTelemetry.Api/Context/Propagation/B3Propagator.cs:23]: Still need this or not?

**Review by reyang — APPROVED** [review]
LGTM

**Review by alanwest — APPROVED** [review]

**Pull request merged**